test/Generators/Microsoft.Gen.Logging/Unit/ParserTests.cs

```diff
@@ -313,23 +299,6 @@ partial class C
             }", inNamespace: false);
     }
 
-    [Theory]
-    [InlineData("_foo")]
-    [InlineData("__foo")]
-    [InlineData("@_foo", "_foo")]
-    public async Task InvalidParameterName(string name, string? template = null)
-    {
-        string source = @$"
-                partial class C
-                {{
-                    [LogMethod(0, LogLevel.Debug, ""M1 {{{template ?? name}}}"")]
-                    static partial void M1(ILogger logger, string /*0+*/{name}/*-0*/);
-                }}
-            ";
-
-        await RunGenerator(source, DiagDescriptors.InvalidLoggingMethodParameterName);
```

Comment:
Same here, just change the assertion that no diags were reported

Reply:
Done.